pkg/catalogv2/system/system.go

```diff
@@ -266,8 +266,12 @@ func (m *Manager) install(namespace, name, minVersion, exactVersion string, valu
 	if err != nil {
 		return err
 	}
-	v := ">=0-a" // latest - this is special syntax to match everything including pre-releases build
+
+	const latestVersionMatcher = ">=0-a" // latest - special syntax to match everything including pre-release builds
```

Comment:
This is good, but I noticed that the comment for this function was not updated to reflect the behavior change.

> // install tries to install a new version of a chart. If the exact version is provided, it will try to install it
// otherwise it will try to install the latest version available. 

Since it was already wrong, you do not have to fix it in this PR. But if you don't mind, we could squeeze it in.

Reply:
Better late than never, and better now than even later :) Thanks for raising this, I've just updated the comment. Feel free to let me know if you had something else in mind.